src/algebra/group/type_tags.lean

```diff
@@ -112,3 +112,12 @@ instance additive.is_add_group_hom [group α] [group β] (f : α → β) [is_gro
 instance multiplicative.is_group_hom [add_group α] [add_group β] (f : α → β) [is_add_group_hom f] :
   @is_group_hom (multiplicative α) (multiplicative β) _ _ f :=
 { map_mul := @is_add_hom.map_add α β _ _ f _ }
+
+/-- Reinterpret `f : α →+ β` as `multiplicative α →* multiplicative β`. -/
+def multiplicative.monoid_hom [add_group α] [add_group β] (f : α →+ β) :
```

Comment:
```suggestion
def add_monoid_hom.as_monoid_hom [add_group α] [add_group β] (f : α →+ β) :
```
I'm not sure I want this change. But it would allow us to use projection notation.
Alternative `add_monoid_hom.multiplicative`...

Reply:
Renamed to `add_monoid_hom.to_multiplicative`.